q2_feature_table/_examples.py

```diff
@@ -0,0 +1,357 @@
+    )
+
+    filtered_seqs, = use.action(
+        use.UsageAction(plugin_id='feature_table', action_id='filter_seqs'),
+        use.UsageInputs(table=feature_table, metadata=sample_metadata,
+                        where=r'[subject]="subject-1" AND [body-site]="gut"'),
+        use.UsageOutputNames(filtered_seqs='filtered_seqs')
+    )
+
+
+def feature_table_filter_seqs_metadata4(use):
+    feature_table = use.init_artifact_from_url(
+        'feature_table', rep_seqs_1_url
+    )
+    sample_metadata = use.init_metadata_from_url(
+        'sample_metadata', moving_pics_md_url
+    )
+
+    filtered_seqs, = use.action(
+        use.UsageAction(plugin_id='feature_table', action_id='filter_seqs'),
+        use.UsageInputs(
+            table=feature_table, metadata=sample_metadata,
+            where=r'[body-site]="gut" OR [reported-antibiotic-usage]="Yes"'),
+        use.UsageOutputNames(filtered_seqs='filtered_seqs')
+    )
+
+
+def feature_table_filter_seqs_metadata5(use):
+    feature_table = use.init_artifact_from_url(
+        'feature_table', rep_seqs_1_url
+    )
+    sample_metadata = use.init_metadata_from_url(
+        'sample_metadata', moving_pics_md_url
+    )
+
+    filtered_seqs, = use.action(
+        use.UsageAction(plugin_id='feature_table', action_id='filter_seqs'),
+        use.UsageInputs(
+            table=feature_table, metadata=sample_metadata,
+            where=r'[subject]="subject-1" AND NOT [body-site]="gut"'),
+        use.UsageOutputNames(filtered_seqs='filtered_seqs')
+    )
+
+
+def feature_table_merge_taxa(use):
+    # TODO: Would probably be better to have two different artifacts here
+    tax1 = use.init_artifact_from_url('tax1', taxonomy_1_url)
+    tax2 = \
+        use.init_artifact_from_url('tax2', taxonomy_1_url)
+
+    merged_taxa, = use.action(
+        use.UsageAction('feature_table', 'merge_taxa'),
+        use.UsageInputs(
+            data=[tax1, tax2]
+        ),
+        use.UsageOutputNames(
+            merged_taxa='merged_taxa'
+        )
+    )
+
+
+def feature_table_filter_samples_min_features(use):
+    feature_table = use.init_artifact_from_url(
+        'feature_table', moving_pics_ft_url
+    )
+
+    filtered_table, = use.action(
+        use.UsageAction(plugin_id='feature_table', action_id='filter_samples'),
+        use.UsageInputs(table=feature_table,
+                        min_features=10),
+        use.UsageOutputNames(filtered_table='filtered_table')
+    )
+
+
+def feature_table_filter_samples_min_frequency(use):
+    feature_table = use.init_artifact_from_url(
+        'feature_table', moving_pics_ft_url
+    )
+
+    filtered_table, = use.action(
+        use.UsageAction(plugin_id='feature_table', action_id='filter_samples'),
+        use.UsageInputs(table=feature_table,
+                        min_frequency=1500),
+        use.UsageOutputNames(filtered_table='filtered_table')
+    )
+
+
+def feature_table_filter_samples_metadata1(use):
```

Comment:
Same thoughts for these subsequent examples as with the `filter_seqs` examples - I'd modify the names to be more descriptive as to how you're filtering.